row_053_views.py

```
@@ -0,0 +1,317 @@
+    if request.method == 'POST':
+        username = request.POST['username']
+        email =  request.POST['email']
+        password = request.POST['password']
+        confirm_password = request.POST['confirm_password']
+        if password == confirm_password:
+
+            if UserModel.objects.filter(email=email).exists():
+                messages.error(request, 'Email already exists')
+                return redirect('register')
+            else:
+                UserModel.objects.create(
```

Ruff (0.13.2)

`UserModel` may be undefined, or defined from star imports

(F405)